### ISSUE TITLE
Types are wrong

### ISSUE THREAD

**Issue by haines** [playwright-community/eslint-plugin-playwright#330]
I get TypeScript errors when attempting to use this plugin. It seems like the types are wrong.

This has a type error (`Property 'configs' does not exist`) but works at runtime:

```ts
import playwrightPlugin from "eslint-plugin-playwright";
import { config as defineConfig } from "typescript-eslint";

export const playwrightConfig = defineConfig(
  playwrightPlugin.configs["flat/recommended"],
);
```

This type-checks successfully but fails at runtime (`TypeError: undefined is not an object (evaluating 'playwrightPlugin.default.configs')`:

```ts
import playwrightPlugin from "eslint-plugin-playwright";
import { config as defineConfig } from "typescript-eslint";

export const playwrightConfig = defineConfig(
  playwrightPlugin.default.configs["flat/recommended"],
);
```

My experience seems to be consistent with the issues reported by "Are The Types Wrong?"

https://arethetypeswrong.github.io/?p=eslint-plugin-playwright%402.0.0

❗️[Incorrect default export](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseExportDefault.md)

The resolved types use export default where the JavaScript file appears to use module.exports =. This will cause TypeScript under the node16 module mode to think an extra .default property access is required, but that will likely fail at runtime. These types should use export = instead of export default.

🎭 [Masquerading as CJS](https://github.com/arethetypeswrong/arethetypeswrong.github.io/blob/main/docs/problems/FalseCJS.md)

Import resolved to a CommonJS type declaration file, but an ESM JavaScript file.

**Comment by mskelton** [comment]
Please share a reproduction code sandbox or repo. The types only matter for the ESM build which is used for the flat config, the commonjs export is only for legacy config files which don't need the types.

**Comment by haines** [comment]
Hi @mskelton, here's a minimal repro: https://github.com/haines/eslint-plugin-playwright-types-are-wrong

**Comment by mskelton** [comment]
Fixed in 4c61256978556570b4f528eb2ba932ebee110609 and faee198b68b1e134ed389b09c0b663011a556d9e

**Comment by haines** [comment]
Thanks!
